apps/web/src/screens/Setting/components/MyWallets/Wallet.tsx

```diff
@@ -1,7 +1,16 @@
-import { FC } from 'react';
+import { type FC } from 'react';
+import { Image, StyleSheet } from 'react-native';
+import CopiedNotification from '@walless/app/components/CopiedNotification';
```

Comment:
1. import from `@walless/app` only, should export the Component in `index.ts` file of the package, and then import it. 
2. Make `CopiedNotification` a generic component that not only handle "copied" message.